src/layers/normalisation.jl

```diff
@@ -44,6 +44,44 @@ end
 
 _testmode!(a::Dropout, test) = (a.active = !test)
 
+"""
+    LRN()
+
+A layer for [Local Response Normalisation](https://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks.pdf).
+This is to be used only for four dimensional arrays. Meant to be used for normalisation
+after the application of ReLU post convolution.
+
+"""
+struct LRNormalisation
+  α
+  β
+  n
+  k
+end
+
+function LRN(α = 10.0^(-4), β = 0.75, n = 5, k = 2)
+  LRNormalisation(α, β, n, k)
+end
+
+function (a::LRNormalisation)(x)
+  y = ones(size(x))
```

Comment:
```suggestion
  y = fill!(similar(x), one(eltype(x)))
```
to work properly with GPU arrays